docs/src/implicit/BDF.md

```diff
@@ -0,0 +1,45 @@
+# OrdinaryDiffEqBDF
+
+Multistep methods, good for large stiff systems.
+
+```@eval
+first_steps = evalfile("./common_first_steps.jl")
+first_steps("OrdinaryDiffEqBDF", "QNDF")
+```
+
+## Full list of solvers
+
+```@docs
+ABDF2
+QNDF
+QNDF1
+QNDF2
+QBDF
+QBDF1
+QBDF2
+MEBDF2
+FBDF
+```
+
+### IMEX Multistep
+
+```@docs
+SBDF
+SBDF2
+SBDF3
+SBDF4
+```
+### IMEX SDIRK
+
+```@docs
+IMEXEuler
+IMEXEulerARK
+```
+
+### DAE
+
+```@docs
+DImplicitEuler
+DABDF2
+DFBDF
+```
```

Comment:
It is a bit weird that BDF houses these exotic solvers.

Reply:
ImplicitEuler is equivalent to BDF 1.  https://en.wikipedia.org/wiki/Backward_differentiation_formula . SBDF are just IMEX BDF methods which treat a part of it explicitly.